pkgs/development/libraries/bzrtp/default.nix

```diff
@@ -7,7 +7,7 @@
 
 stdenv.mkDerivation rec {
   pname = "bzrtp";
-  version = "4.4.9";
+  version = "4.4.34";
 
   src = fetchFromGitLab {
     domain = "gitlab.linphone.org";
```

Comment:
You missed updating the hash.

Reply:
Well spotted! But, actually, I didn't.. The tags just seem to point to the same commit. :open_mouth: 

https://gitlab.linphone.org/BC/public/bzrtp/-/tags

And it looks like there really hasn't been many commits even though the the tag has been bumped a lot:

https://gitlab.linphone.org/BC/public/bzrtp/-/commits/master/

So, I suppose they (Linphone) just tag all the projects so that their versions are in sync even if some projects haven't had any changes they want to release yet. Or that's my interpretation. Weird.